NWNXLib/API/Constants/Misc.hpp

```diff
@@ -288,4 +288,19 @@ namespace AssociateType
     }
 }
 
+namespace MetaMagicType
+{
+    enum TYPE
+    {
+        None            = 0x00,
+        Empower         = 0x01,
+        Extend          = 0x02,
+        Maximize        = 0x04,
+        Quicken         = 0x08,
+        Silent          = 0x10,
+        Still           = 0x20,
+        Any             = 0xFF,
+    };
+}
```

Comment:
Can you add the MIN/MAX/ToString() stuff for posterity? Like the other constants have

Reply:
Done!